next.config.js

```diff
@@ -0,0 +1,9 @@
+const withImages = require('next-images')
+
+module.exports = {
+   ...withImages(),
+   future: {
+      webpack5: true,
+   },
```

Comment:
@cpaule1811 what's this setting for, the withImages.

Reply:
This is an image loader, this version of next didn't have inbuilt image support from the looks of things.